data/json/mapgen/abandoned01.json

```diff
@@ -1,696 +1,614 @@
+        "^..z..     ...        [.",
+        ".....H6     .         [.",
+        ".##0###2###3+###2###2##.",
+        ".#       #    3  66  >#.",
+        ".#   .   +   7#  66  .z.",
+        ".#  ...  #   7#      ..^",
+        ".#########   7#       #.",
+        ".#___ 6   .. 7#####+###S",
+        ".2_=_     ..... #9   9#.",
+        ".2___   ...ww^. #9   9#^",
+        ".# H     ..wWw  #9   9#.",
+        "^#####+#..S ... ###+###.",
+        ".3..     ..#   z  +  ;#.",
+        ".#.^..   ..#@@    #A   z",
+        ".#   ...   #@@   :# 88#.",
+        "...0##.^.#####22#.#####~",
+        ".zw.......zz...z....^...",
+        ".S.ww^...T......Tz.z.S..",
+        "..^...T......z.^..^.z.^."
+        ],
+        "terrain": {
+          ".": [ "t_grass", "t_grass","t_grass", "t_dirt", "t_dirt" ],
+          "#": "t_wall_log",
+          "2": "t_window_domestic",
+          "0": "t_window_empty",
+          "3": "t_door_b",
+          "w": "t_water_sh",
+          "W": "t_water_dp",
+          " ": "t_floor",
+          "+": "t_door_c",
+          "=": "t_rock_floor",
+          "z": "t_underbrush",
+          "S": "t_shrub_strawberry",
+          "^": "t_tree_young",
+          "T": "t_tree",
+          "{": "t_door_glass_c",
+          "[": "t_fence_v",
+          "_": "t_rock_floor",
+          "~": "t_fence_h"
+        },
+        "furniture": {
+          "6": "f_table",
+          "7": "f_bookcase",
+          "8": "f_bathtub",
+          "9": "f_rack",
+          ":": "f_dresser",
+          "=": "f_woodstove",
+          ">": "f_counter",
+          "?": "f_sofa",
+          "@": "f_bed",
+          "A": "f_sink",
+          "B": "f_chair",
+          "C": "f_desk",
+          "D": "f_trashcan",
+          "E": "f_cupboard",
+          "F": "f_fridge",
+          "H": "f_armchair",
+          "G": "f_oven"
+        },
+        "toilets": {
+          ";": { }
+        },
+        "set": [
+          { "point": "bash", "x": 7, "y":  7},
+          { "point": "bash", "x": 3, "y":  20},
+          { "point": "bash", "x": [ 14, 22], "y":  4, "repeat": [ 3, 5] },
+          { "point": "bash", "x": [ 14, 22], "y":  7, "repeat": [ 3, 5] },
+          { "point": "bash", "x": [ 11, 12], "y":  7, "repeat": [ 0, 2] },
+          { "point": "bash", "x": [ 1, 9], "y":  11, "repeat": [ 0, 8] },
+          { "point": "bash", "x": [ 13, 15], "y":  20, "repeat": [ 0, 4] },
+          { "point": "bash", "x": [ 17, 21], "y":  [ 13, 15], "repeat": [ 4, 10] },
+          { "point": "terrain", "id": "t_dirt", "x": 0, "y": [ 0, 23 ], "repeat": [ 5, 10 ] },
+          { "point": "terrain", "id": "t_dirt", "x": 0, "y": [ 0, 23 ], "repeat": [ 5, 10 ] },
+          { "point": "terrain", "id": "t_dirt", "x": 23, "y": [ 0, 23 ], "repeat": [ 5, 10 ] },
+          { "point": "terrain", "id": "t_dirt", "x": [ 0, 23 ], "y": [ 21,23 ], "repeat": [ 5, 8 ] },
+          { "point": "terrain", "id": "t_dirt", "x": [ 0, 14 ], "y": [ 0,3 ], "repeat": [ 8, 10 ] }
```

Comment:
must have missed it under all the 'bash'ing.  Not completely sure what bash does, but I can guess it simulates an attack on the present terrain/funriture